polybar-scripts/docker-container/docker-container.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Use `#!/bin/sh` for posix compliance.

Reply:
fixed with [#9e3634d](https://github.com/polybar/polybar-scripts/commit/9e3634d6bbc35f8f92321898d1daa85ef7ce5f76)